iceoryx_dds/CMakeLists.txt

```diff
@@ -38,13 +36,13 @@ setup_package_name_and_create_files(
 #
 ########## feature flags ##########
 #
-option(USE_CYCLONE_DDS "Bind to CycloneDDS implementation" on)
+option(USE_CYCLONE_DDS "Bind to CycloneDDS implementation" ON)
 
 if(USE_CYCLONE_DDS)
     message(INFO " Using CycloneDDS stack")
     find_package(CycloneDDS CONFIG REQUIRED)
     find_package(CycloneDDS-CXX CONFIG REQUIRED)
-endif()
+endif(USE_CYCLONE_DDS)
```

Comment:
the argument is `endif` is optional and supported for backward compatibility only. I would suggest to not increase this style.
 
https://cmake.org/cmake/help/latest/command/endif.html